setup/templates/steps/compose/03_expose.html

```diff
@@ -40,13 +40,16 @@
   <input class="form-control" type="text" name="subnet6" required value="{{ subnet6 }}:beef::/64">
 </div>
 
-<p>The unbound resolver enables Mailu to do DNSSEC verification, DNS root lookups and caching. This also helps the antispam service not to get blocked by the public or ISP DNS servers.</p>
 <div class="form-check form-check-inline">
+  <p>The unbound resolver enables Mailu to do DNSSEC verification, DNS root lookups and caching. This also helps the antispam service not to get blocked by the public or ISP DNS servers.</p>
   <label class="form-check-label">
     <input class="form-check-input" type="checkbox" name="resolver_enabled" value="true" checked>
```

Comment:
Should get a property id="resolver_enabled" which can be used in render.js for showing the warning message element.